professorabhay/komiser
# Update policy.json

Update Komiser AWS IAM Policy #852



## How to Test

Add this Policy to an IAM role and run komiser with that role, and you would  not see any fetch permissions errors which shown previously.
